zproject/dev_settings.py

```diff
@@ -212,3 +212,13 @@
 # or others involving new bouncer-side logic.
 # DEVELOPMENT_DISABLE_PUSH_BOUNCER_DOMAIN_CHECK = True
 # PUSH_NOTIFICATION_BOUNCER_URL = "http://localhost:9991"
+
+# You can override any of these settings locally by creating a
+# zproject/custom_dev_settings.py. This is useful for settings that
+# vary across development environments and should not be in version control.
+try:
+    from zproject.custom_dev_settings import *  # noqa: F403
+except ImportError:
+    # Create file if it doesn't exist
+    with open(os.path.join(DEPLOY_ROOT, "zproject/custom_dev_settings.py"), "w"):
+        pass
```

Comment:
Why do you create this if it doesn't exist rather than doing nothing?

I ask because creating an extra file will make using old branches that predate this commit show it an untracked file when moving around.